web_src/js/globals.js

```diff
@@ -0,0 +1,5 @@
+import $ from 'jquery';
+import htmx from 'htmx.org/dist/htmx.esm.js';
+
+window.$ = window.jQuery = $; // eslint-disable-line no-jquery/variable-pattern
+window.htmx = htmx;
```

Comment:
```suggestion
import jquery from 'jquery';
import htmx from 'htmx.org/dist/htmx.esm.js';

window.$ = window.jQuery = jquery;
window.htmx = htmx;
```

And the old `jquery.js` should be removed?